src/cmdhandler/command_handler.cpp

```diff
@@ -370,28 +440,218 @@ std::string command_handler::collector_system(const std::vector<std::string>& to
+        }else if(variable == "camera_options"){
+            std::stringstream response;
+            response << variable << ": ";
+
+            for(auto const &option : current_state.camera.camera_options){
+                response << "\n    " << option.first << ": " << std::boolalpha << option.second;
+            }
+
+            return response.str();
+        }
+
+        return "variable '"+variable+"' does not exist";
+    }else if(tokens[0] == "delete"){
+        if(tokens.size() < 3){
+            return "please provide a variable to delete\n    ex: delete camera url";
+        }
+
+        std::string variable = tokens[2];
+
+        if(variable == "url"){
+            current_state.camera.url = "";
+        }else if(variable == "camera_matrix"){
+            current_state.camera.camera_matrix.release();
+        }else if(variable == "distortion_matrix"){
+            current_state.camera.distortion_matrix.release();
+        }else if(variable == "marker_dictionary"){
+            current_state.camera.marker_dictionary = 0;
+        }else if(variable == "camera_options"){
+            current_state.camera.camera_options.clear();
+        }else{
+           return "variable '"+variable+"' does not exist";
+        }
+
+        return "'"+variable+"' variable has been deleted";
+    }else{
+        return "command '"+tokens[0]+"' not valid for target system '"+tokens[1]+"'";
+    }
+}
+
 /**
  * Display available target systems/commands to user
  *
  * @return help string
  */
 std::string command_handler::help_command(){
     std::string response = "current target systems:\n";
-    response += "    robot, state, collector\n\n";
+    response += "    robot, state, collector, camera\n\n";
 
     response += "for the 'robot' system you can use the commands:\n";
     response += "    get, set, list, delete\n";
     response += "ex: 'get robot robot_1' or 'list robot' or 'set robot robot1 1,2,3,4' or 'delete robot robot1'\n\n";
 
     response += "for the 'state' system you can use the commands:\n";
-    response += "    save, load, delete\n";
-    response += "ex: 'save state config1' or 'load state config1' or 'delete state config1'\n";
+    response += "    save, load, delete, list\n";
+    response += "ex: 'save state config1' or 'load state config1' or 'delete state config1' or 'list state'\n";
     response += "NOTE: keyword 'current' is used for clearing current state ('delete state current'). Cannot save a state with the name 'current'\n\n";
 
     response += "for the 'collector' system you can use the commands:\n";
     response += "    get, set, list, delete\n";
     response += "ex: 'get collector gcs' or 'list collector' or 'set collector gcs 127.0.0.1 53' or 'delete collector gcs'\n\n";
 
-    response += "intended usage for each target system will be clarified if used incorrectly.\n\n";
+    response += "for the 'camera' system you can use the commands:\n";
+    response += "    get, set, list, delete\n";
```

Comment:
It might be good to specify that `list camera` lists the current variables set within camera, not a list of available cameras that set/get or something similar can be used on.